lib/src/common/exception.dart

```diff
@@ -258,7 +258,7 @@ WebDriverException getExceptionFromW3cResponse({
   dynamic jsonResp,
 }) {
   if (jsonResp is Map && jsonResp.keys.contains('value')) {
-    final value = jsonResp['value'];
+    final value = jsonResp['value'] as Map<String, dynamic>;
```

Comment:
As a nit, I think we're going to start preferring `Map<String, Object?>` here instead of `Map<String, dynamic>`.